html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
+<iframe data-src="about:blank#foo" src="support/blank.htm?src" loading="lazy"></iframe>
+<script>
+const iframe = document.querySelector('iframe');
+let pushStateSuccess = true;
+try {
+  // Should have no effect on lazy-loading.
+  // Per https://html.spec.whatwg.org/#can-have-its-url-rewritten
+  // only the fragment can be changed for about: URLs.
+  iframe.contentWindow.history.pushState(null, "", iframe.dataset.src);
+} catch(ex) {
+  replaceStateSuccess = false;
```

Comment:
Should be `pushStateSuccess`, oops

Reply:
I think this can be resolve now given the latest commit? I'll resolve this.